File: node/handler.js
const AWS = require('aws-sdk')
const db = new AWS.DynamoDB.DocumentClient()
const TableName = 'tablename'

module.exports.hello = async (event, context) => {
  const items = await db.scan({
    TableName
  }).promise()

  return {
    statusCode: 200,
    body: JSON.stringify({
      message: items,
      input: event,
    })
  }
}

File: python/handler.py
import boto3

dynamodb = boto3.resource('dynamodb')
table = dynamodb.Table('tablename')


def get_person(id):
    response = table.get_item(Key={'id': id})

    return response['Item']

def get_persons():
    response = table.scan()

    return response['Items']

def hello(event, context):
    return get_person(event['id']) if event['id'] else get_persons()
